include/sdf/Capsule.hh

```diff
@@ -72,6 +75,12 @@ namespace sdf
     /// \return A reference to a gz::math::Capsuled object.
     public: gz::math::Capsuled &Shape();
 
+    /// \brief Calculate and return the Mass Matrix values for the Capsule
+    /// \param[in] density Density of the capsule in kg/m^3
```

Comment:
```suggestion
    /// \param[in] _density Density of the capsule in kg/m^3
```

Reply:
Corrected in [ac943c6](https://github.com/gazebosim/sdformat/pull/1299/commits/ac943c652f84ff23acc602fd13f00f9d9a32f2a7)